web/themes/custom/pineapple/sass/base/forms/_form-items.scss

```diff
@@ -0,0 +1,55 @@
+.user-register-form,
+.user-login-form {
```

Comment:
There is gonna be other forms in the near future do you also want them to have this styling?

Reply:
understood